README.md

```diff
@@ -63,7 +67,7 @@ $ pipenv sync
 $ MOLECULE_DISTRO=(debian8|debian9|debian10) pipenv run molecule test
```

Comment:
I think that debian8|debian9|debian10 values are no longer supported

Reply:
You're right, I have removed them.